Dockerfile

```diff
@@ -51,8 +58,9 @@ ENV UMPIRE_VERSION 0.5.3
 # Install umpire
 RUN pip2.7 install umpire==${UMPIRE_VERSION}
 
-#Install go
-RUN yum install -y golang
+# Install golang 1.12
+RUN wget https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz -O /tmp/go1.12.1.linux-amd64.tar.gz
```

Comment:
1.12.3 is the latest stable (https://dl.google.com/go/go1.12.3.linux-amd64.tar.gz)